### PR TITLE
feat: python static file

### PR DESCRIPTION
**Pull request by ScmTble**

### CODE REVIEW

**Comment by pan93412** [comment]
Better to seperate to two commits. Steps:

1. `git rebase -i main`
2. Mark your commit as `edit`
3. Open your VS Code.
4. Type `git reset HEAD~` in your terminal to reset the commit.
5. Commit your `java` folder first with the commit message: `fix(planner/java): Skip Java test`
6. Commit your `python` folder second with the commit message: `feat(planner/python): Add static file support`
7. Run `git push -f` to force push your changes.

File: internal/python/python.go [hunk 1]
```diff
@@ -10,17 +10,42 @@ func GenerateDockerfile(meta types.PlanMeta) (string, error) {
...
+&& echo "\
+server { \
+        listen 8080; \
+        location / { \
+			proxy_pass              http://127.0.0.1:8000; \
+			proxy_set_header        Host \$host; \
+		} \
+		location /static { \
+			autoindex on; \
+			alias /app/static/ ; \` + `
```
Comment (pan93412) [inline internal/python/python.go:30]: This backslash in the line trail may add an unexpected newline.

File: internal/python/plan.go [hunk 2]
```diff
@@ -344,7 +344,7 @@ func determineStartCmd(ctx *pythonPlanContext) string {
 	framework := DetermineFramework(ctx)
 	pm := DeterminePackageManager(ctx)
 	var commandSegment []string
-
+	commandSegment = append(commandSegment, "/usr/sbin/nginx && ")
```
Comment (pan93412) [inline internal/python/plan.go:347]: It should be only added when wsgi is specified.

File: internal/python/plan.go [hunk 3]
```diff
@@ -318,7 +318,7 @@ func determineInstallCmd(ctx *pythonPlanContext) string {
 }
 
 func determineAptDependencies(ctx *pythonPlanContext) []string {
-	deps := []string{"build-essential"}
+	deps := []string{"build-essential", "nginx"}
```
Comment (pan93412) [inline internal/python/plan.go:321]: Is this dependency necessary in every Python applications?

**Review by pan93412 — CHANGES_REQUESTED** [review]

File: internal/python/plan.go (hunk 3 above)
Comment (ScmTble) [reply internal/python/plan.go:321]: no, only wsgi ,i will change

**Pull request merged**